src/daemon/Dockerfile

```diff
@@ -0,0 +1,59 @@
+# CEPH DAEMON IMAGE
+
+FROM MAKEFILE_REPLACE_WITH_DAEMON_BASE_IMAGE
+MAINTAINER __DOCKERFILE_MAINTAINER__
+
+#======================================================
+# Install ceph and dependencies, and clean up
+#======================================================
+__DOCKERFILE_PREINSTALL__
+
+# Escape char after immediately after RUN allows comment in first line
+RUN \
+    # Install all components for the image, whether from packages or web downloads.
+    # Typical workflow: add new repos; refresh repos; install packages; package-manager clean;
+    #   download and install packages from web, cleaning any files as you go.
+    __DOCKERFILE_INSTALL__ && \
+    # Clean container, starting with record of current size
+    INITIAL_SIZE=$(du -sm / 2>/dev/null | awk '{print $1}') && \
+    #
+    #
+    # Clean daemon-specific files
+    __DOCKERFILE_CLEAN_DAEMON__ && \
+    # Perform any final cleanup actions like package manager cleaning, etc.
+    __DOCKERFILE_POSTINSTALL_CLEANUP__ && \
+    # Clean common files like /tmp, /var/lib, etc.
+    __DOCKERFILE_CLEAN_CONTAINER_COMMON__ && \
+    #
+    #
+    # Report size savings
+    FINAL_SIZE=$(du -sm / 2>/dev/null | awk '{print $1}') && \
+    REMOVED_SIZE=$((INITIAL_SIZE - FINAL_SIZE)) && \
+    echo "Cleaning process removed ${REMOVED_SIZE}MB" && \
+    echo "Dropped container size from ${INITIAL_SIZE}MB to ${FINAL_SIZE}MB"
+
+#======================================================
+# Add ceph-container files
+#======================================================
+
+# Add s3cfg file
+ADD s3cfg /root/.s3cfg
+
+# Add templates for confd
+ADD ./confd/templates/* /etc/confd/templates/
+ADD ./confd/conf.d/* /etc/confd/conf.d/
+
+# Add bootstrap script, ceph defaults key/values for KV store
+ADD *.sh ceph.defaults check_zombie_mons.py ./osd_scenarios/* entrypoint.sh.in disabled_scenario /
+
+# Modify the entrypoint
+RUN bash "/generate_entrypoint.sh" && \
+  rm -f /generate_entrypoint.sh && \
+  bash -n /*.sh
+
+# Add volumes for Ceph config and data
+VOLUME ["/etc/ceph","/var/lib/ceph", "/etc/ganesha"]
```

Comment:
@BlaineEXE , @leseb told me this wasn't required anymore.

Reply:
If that's true, that is a different PR.